langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiImageModel.java

```diff
@@ -217,6 +360,28 @@ private Value prepareParameters(int n) throws InvalidProtocolBufferException {
             paramsMap.put("language", this.language);
         }
 
+        if (this.aspectRatio != null) {
+            paramsMap.put("aspectRatio", this.aspectRatio.getRatio());
+        }
+
+        if (this.mimeType != null) {
+            Map<String, Object> outputOptions = new HashMap<>();
+            outputOptions.put("mimeType", this.mimeType.getMimeType());
+            if (this.mimeType == MimeType.JPEG && this.compressionQuality != null) {
+                outputOptions.put("compressionQuality", this.compressionQuality);
+            }
+
+            paramsMap.put("outputOptions", outputOptions);
+        }
+
+        if (this.personGeneration != null) {
+            paramsMap.put("personGeneration", this.personGeneration);
+        }
+
+        if (this.watermark != null) {
+            paramsMap.put("addWatermark", this.watermark);
```

Comment:
```suggestion
            paramsMap.put("addWatermark", this.addWatermark);
```
maybe `addWatermark` to keep naming in sync?